From 86b6ae81e020b228ee5b3b235abc496f5c80b52d Mon Sep 17 00:00:00 2001
From: Andreas Schneider <asn@cryptomilk.org>
Date: Mon, 22 Jan 2018 07:47:08 +0100
Subject: [PATCH 15/18] storage: Set all sdcards to visible

If they are visible to apps the media scanner will scan them.

Change-Id: Icceceafc8469b6b9d1c0e72f8425d7e38ae4a10b
(cherry picked from commit 48dc2e83accb27b99c8a713b95988c25111c392b)
---
 data/etc/privapp-permissions-platform.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/etc/privapp-permissions-platform.xml b/data/etc/privapp-permissions-platform.xml
index 1836434949f8..5914defd04a8 100644
--- a/data/etc/privapp-permissions-platform.xml
+++ b/data/etc/privapp-permissions-platform.xml
@@ -213,6 +213,7 @@ applications that come with the platform
         <permission name="android.permission.MANAGE_USERS"/>
         <permission name="android.permission.USE_RESERVED_DISK"/>
         <permission name="android.permission.WRITE_MEDIA_STORAGE"/>
+        <permission name="android.permission.READ_EXTERNAL_STORAGE"/>
     </privapp-permissions>
 
     <privapp-permissions package="com.android.providers.telephony">
-- 
2.17.1

